jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-6a.yaml

```diff
@@ -83,25 +83,41 @@
       - string:
           name: cloudsource
           default: develcloud$cloud_version
+
       - string:
           name: TESTHEAD
           default: "1"
           description: if non-empty, test latest version from Devel:Cloud Stagin
+
       - string:
           name: hacloud
           default: "1"
           description: By default we do not want HA configured and installed
+
       - string:
           name: clusterconfig
           default: services=2
           description: HA configuration for clusters. Make sense only if hacloud=1
+
+      - string:
+          name: networkingplugin
+          default: openvswitch
+          description: networking plugin to be used by neutron
+
+      - string:
+          name: networkingmode
+          default: vlan
+          description: networking mode to be used by neutron
```

Comment:
Add the available options here as well please, should be clear which options fit to which networking plugin.